requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Includes https://github.com/thanethomson/statik/pull/78
+git+https://github.com/jayvdb/statik@coala
```

Comment:
this can be removed once I merge https://github.com/thanethomson/statik/pull/78 , which would make it easier for students to use statik